tools/genesis/main.go

```diff
@@ -77,16 +56,49 @@ func main() {
 	defer xls.Close()
 	allocationRows := loadRows(xls, workbook.Allocations)
 	multisigRows := loadRows(xls, workbook.MultisigDefinitions)
+	depositOffersRows := loadRows(xls, workbook.DepositOffers)
 
-	allocations, err := parseAllocations(allocationRows)
+	allocations := parseAllocations(allocationRows)
 	fmt.Println("Loaded allocations", len(allocations), "err", err)
-	multisigs, err := parseMultiSigGroups(multisigRows, allocations)
+	// Pick the max start offset to delay deposit offers end
+	maxStartOffset := uint64(0)
+	for _, allocation := range allocations {
+		if allocation.TokenDeliveryOffset > maxStartOffset {
+			maxStartOffset = allocation.TokenDeliveryOffset
+		}
+	}
+
+	// Set ID on DepositOffers
+	offersMap := make(map[string]genesis.UnparsedDepositOffer)
+	// TODO: make this generateDepositOffers to match the other data processing
+	depositOffers := []genesis.UnparsedDepositOffer{}
+	oo := parseDepositOfferRows(depositOffersRows)
```

Comment:
It would be better to have something more than "oo" as a variable name as I just can't deduct any meaning out of that abbr.